session/handler.go

```diff
@@ -194,6 +198,89 @@ func (h *Handler) logout(w http.ResponseWriter, r *http.Request, ps httprouter.P
 	w.WriteHeader(http.StatusAccepted)
 }
 
+// swagger:parameters adminIdentitySession
+// nolint:deadcode,unused
+type adminIdentitySession struct {
+	// ID is the identity's ID.
+	//
+	// required: true
+	// in: path
+	ID string `json:"id"`
+}
+
+// swagger:model successfulAdminIdentitySession
+// nolint:deadcode,unused
+type AdminIdentitySessionResponse struct {
+	// The Session Token
+	//
+	// This field is only set when the session hook is configured as a post-registration hook.
+	//
+	// A session token is equivalent to a session cookie, but it can be sent in the HTTP Authorization
+	// Header:
+	//
+	// 		Authorization: bearer ${session-token}
+	//
+	// The session token is only issued for API flows, not for Browser flows!
+	Token string `json:"session_token"`
+
+	// The Session
+	//
+	// The session contains information about the user, the session device, and so on.
+	//
+	// required: true
+	Session *Session `json:"session"`
+
+	// The Identity
+	//
+	// The identity that just signed up.
+	//
+	// required: true
+	Identity *identity.Identity `json:"identity"`
+}
+
+// swagger:route GET /sessions/identity/{id} v0alpha1 adminIdentitySession
+//
+// Calling this endpoint issues session for given identity.
+//
+// This endpoint is useful for:
+//
+// - Issuing session or session token for a given identity without authenticating
+//
+//     Schemes: http, https
+//
+//     Security:
+//       oryAccessToken:
+//
+//     Responses:
+//       200: successfulAdminIdentitySession
+//       404: jsonError
+//       500: jsonError
+func (h *Handler) session(w http.ResponseWriter, r *http.Request, ps httprouter.Params) {
```

Comment:
This seems to be GoLang's code style, but darn I hate these non-descriptive variable names.